websites/R/Repeat/dist/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+  "author": {
+    "name": "Callum",
+    "id": "506899274748133376"
+  },
+  "service": "Repeat",
+  "description": {
+    "en": "Repeating your favourite tunes daily."
+  },
+  "url": [
+    "www.repeatradio.net",
+    "repeatradio.net",
+    "repeat.pw"
+  ],
+  "version": "2.5.3",
+  "logo": "https://repeatradio.net/staff/avatars/default.png",
+  "thumbnail": "https://repeatradio.net/staff/covers/presets/1.png",
```

Comment:
upload to imgur

Reply:
self hosted not needed

Reply:
It is still not allowed, please upload to imgur.

Reply:
"or on a CDN that gives you permissions to change image files" this is our cdn, and we wont be changing the image file ever